pkg/models/dashboard_thumbs.go

```diff
@@ -123,6 +124,7 @@ type FindDashboardThumbnailCountCommand struct {
 
 type FindDashboardsWithStaleThumbnailsCommand struct {
 	IncludeManuallyUploadedThumbnails bool
+	IncludeThumbnailsWithEmptyDsUids  bool
```

Comment:
This settings seems a bit excessive -- it looks like this is really "is search enabled"

Reply:
Right, at some point the `isSearchEnabled` has to be translated into `include thumbnails with empty ds uids`, and here its done at the service rather than at the DB level, to keep the latter dumb/business logic free. I have done it out of principle more than anything - the condition here is super simple so I can move it if it makes more sense that way